libs/tree-sitter-wing/grammar.js

```diff
@@ -515,7 +516,44 @@ module.exports = grammar({
     await_expression: ($) => prec.right(seq("await", $.expression)),
     defer_expression: ($) => prec.right(seq("defer", $.expression)),
     parenthesized_expression: ($) => seq("(", $.expression, ")"),
+    
+    json_literal: ($) => choice(
+      seq(field("type", choice($.json_container_type)), field("element", $._json_value)),
```

Comment:
`$.json_container_type` is already wrapped in a `choice`, this is redundant.
```suggestion
      seq(field("type", $.json_container_type), field("element", $._json_value)),
```